server/src/test/resources/allowed_queries_with_path_type.json

```diff
@@ -0,0 +1,20 @@
+{
+  "entries": [
+    {
+      "path": "Observation",
+      "queryParams": {
```

Comment:
Do we need to keep empty `queryParams`? Or does it default to an empty map if we remove these? If yes, consider removing them.

Reply:
This is set to null in the config if we remove this leading to NPE. 
I was thinking of fixing that in a different PR.